firmware/powerboard2/code/app/housingMonitor.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * housingMonitor.c
+ *
+ *  Created on: Sep 7, 2020
+ *      Author: Shuhang Tan
+ */
+
+#include "housingMonitor.h"
+#include "ADC.h"
+
+
+#define threshold (2048)
+housingMonitor_housingStatus_E HStatus;

```

Comment:
Don't put global variables outside a struct. Create a struct for this file to put all global variables in. It also need to be static.
```
typedef struct
{
    housingMonitor_housingStatus_E housingStatus;
} housingMonitor_data_S;

static housingMonitor_data_S housingMonitor_data;
```